### DISCUSSION TITLE
Can quadrant be used in a distributed cluster? On several different servers if there is more data than RAM om one server.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [qdrant/qdrant#2982] [General]
Can quadrant be used in a distributed cluster? On several different servers if there is more data than RAM om one server.
Is there a guide on how to use it such a way?

**Comment by timvisee** [comment]
Yes, Qdrant can be run in a cluster to support huge scales.

Please read through our distributed deployment guide: https://qdrant.tech/documentation/guides/distributed_deployment/
